tools/influx_si/macros.xml

```diff
@@ -0,0 +1,265 @@
+              <param argument="--emu" type="boolean" checked="false" truevalue="TRUE" falsevalue="FALSE" label="--emu" optional="true" help="simulate labeling in EMU approach" />
+              <param argument="--irand" type="boolean" checked="false" truevalue="TRUE" falsevalue="FALSE" label="--irand" optional="true" help="ignore initial approximation for free parameters (free fluxes and metabolite concentrations) from the FTBL file or from a dedicated file (cf --fseries and --iseries option) and use random values drawn uniformly from [0,1] interval" />
+              <param argument="--sens" type="text" value="" label="--sens" optional="true" help="sensitivity method: SENS can be &#x27;mc[=N]&#x27;, mc stands for Monte-Carlo. N is an optional number of Monte-Carlo simulations. Default for N: 10" />
+              <param argument="--cupx" type="float" min="0" max="1" value="" label="--cupx" optional="true" help="upper limit for reverse fluxes. Must be in interval [0, 1]. Default: 0.999" />
+              <param argument="--cupn" type="float" min="0" value="" label="--cupn" optional="true" help="absolute limit for net fluxes: -cupn &lt;= netflux &lt;= cupn. Must be non negative. Value 0 means no limit. Default: 1.e3" />
+              <param argument="--cupp" type="float" min="0" value="" label="--cupp" optional="true" help="upper limit for metabolite pool. Default: 1.e5" />
+              <param argument="--clownr" type="float" min="0" value="" label="--clownr" optional="true" help="lower limit for not reversible free and dependent fluxes. Zero value (default) means no lower limit" />
+              <param argument="--cinout" type="float" min="0" value="" label="--cinout" optional="true" help="lower limit for input/output free and dependent fluxes. Must be non negative. Default: 0" />
+              <param argument="--clowp" type="float" min="0" value="" label="--clowp" optional="true" help="lower limit for free metabolite pools. Must be positive. Default 1.e-8" />
+              <param argument="--np" type="float" min="0" value="" label="--np" optional="true" help="When integer &gt;= 1, it is a number of parallel subprocesses used in Monte-Carlo (MC) simulations or for multiple FTBL inputs. When NP is a float number between 0 and 1, it gives a fraction of available cores (rounded to closest integer) to be used. Without this option or for NP=0, all available cores in a given node are used for MC simulations." />
+              <param argument="--ln" type="boolean" checked="false" truevalue="TRUE" falsevalue="FALSE" label="--ln" optional="true" help="Least norm solution is used for increments during the non-linear iterations when Jacobian is rank deficient" />
+              <param argument="--sln" type="boolean" checked="false" truevalue="TRUE" falsevalue="FALSE" label="--sln" optional="true" help="Least norm of the solution of linearized problem (and not just of increments) is used when Jacobian is rank deficient" />
```

Comment:
There is a nice feature with booleans that you can use. If you write truevalue="--sin" falsevalue="" ... then you can use in the command section just `$sin` without andy `if/endif`